testsuite/python_support/utils.py

```diff
@@ -264,9 +264,13 @@ def run(*argv):
         with open('dune-project', 'w') as f:
```

Comment:
The commit content and its message disagree with each other. :-) I suggest to split this commit in three:

* add a Valgrind suppression file for OCaml programs
* python_support/utils.py: build and run OCaml programs separately
* ocaml_api/general: enhance to test BigInt bindings

Reply:
Thanks a lot for the extra explanations in the new commits, they allowed me to understand changes much better. :-)